tests/generateTestCase.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+\error_reporting(E_ALL | E_STRICT);
+\ini_set('display_errors', '1');
+
+//Composer
+require \dirname(__DIR__) . '/vendor/autoload.php';
+
+$dir = __DIR__ . '/ORM/Functional/Driver/Common/Integration';
+$caseTemplateDir = __DIR__ . '/ORM/Functional/Driver/Common/Integration/CaseTemplate';
+
+$cases = 0;
+
+foreach (\scandir($dir) as $dirName) {
+    if (\str_starts_with($dirName, 'Case') && $dirName !== 'CaseTemplate') {
+        $cases++;
+    }
+}
+
+$caseName = 'Case' . $cases + 1;
```

Comment:
That approach doesn't guarantee the `$caseName` will contain unique name with next number